clang/test/Misc/diag-template-diffing-cxx11.cpp

```diff
@@ -265,14 +265,14 @@ int k9 = f9(V9<double>());
 // CHECK-ELIDE-TREE:   S9<
 // CHECK-ELIDE-TREE:     [2 * ...],
 // CHECK-ELIDE-TREE:     U9<
-// CHECK-ELIDE-TREE:       [(no qualifiers) != const] double>>
```

Comment:
It is not obvious why this changed, can you explain?

Reply:
Yeah, with the Subst type node, you get difference when printing  if the node is top-level qualified, versus when it is bottom-qualified. So you can get harmless differences in the order const appears, and in this case here differences in the order you see qualifiers as you single step desugar the types.

The tree-differ could be improved to remove these inconsistencies, but given the harmless nature of the effect we don't need to do it now.